inference-engine/src/mkldnn_plugin/ngraph_transformations/reshape_1d_ops.cpp

```diff
@@ -133,73 +170,79 @@ ngraph::matcher_pass_callback get_callback() {
         new_ops.push_back(last.get_node_shared_ptr());
 
         // if convolution is followed by add we need to replace add before output reshape to fuse conv+bias on plug-in side
-        std::shared_ptr<ngraph::Node> addToReplace = nullptr;
-        std::shared_ptr<ngraph::Node> reshapedAdd = nullptr;
-        ngraph::NodeVector biasOps;
+        std::shared_ptr<ngraph::Node> add_to_replace = nullptr;
+        std::shared_ptr<ngraph::Node> reshaped_add = nullptr;
+        ngraph::NodeVector bias_ops;
         if (std::dynamic_pointer_cast<ngraph::opset1::Convolution>(node) || std::dynamic_pointer_cast<ngraph::opset1::GroupConvolution>(node)) {
-            ngraph::Shape expectedShape = ngraph::Shape(node->get_output_shape(0).size(), 1);
-            expectedShape[1] = node->get_output_shape(0)[1];
-            const auto dstNodes = node->get_output_target_inputs(0);
-            if (dstNodes.size() == 1) {
-                addToReplace = dstNodes.begin()->get_node()->shared_from_this();
-                if (std::dynamic_pointer_cast<ngraph::opset1::Add>(addToReplace) &&
-                    std::dynamic_pointer_cast<ngraph::opset1::Constant>(addToReplace->input(1).get_source_output().get_node_shared_ptr()) &&
-                        addToReplace->get_input_shape(1) == expectedShape) {
-                    ngraph::Shape newBiasShape(addToReplace->get_input_shape(1));
-                    newBiasShape.push_back(1);
-                    auto newBias = ngraph::op::util::reshapeTo(addToReplace->input_value(1), newBiasShape);
-                    reshapedAdd = std::make_shared<ngraph::opset1::Add>(last, newBias);
-                    reshapedAdd->set_friendly_name(addToReplace->get_friendly_name() + "/new");
-                    biasOps.push_back(newBias);
-                    biasOps.push_back(reshapedAdd);
+            const auto dst_nodes = node->get_output_target_inputs(0);
+            auto channel = node->get_output_partial_shape(0)[1];
+            if (dst_nodes.size() == 1 && channel.is_static()) {
```

Comment:
nit: Please add a comment here that it's okay not to reshape biases

Reply:
Done, thanks